common/entity/Person.ts

```diff
@@ -66,6 +66,16 @@ export abstract class Person {
         default: false,
     })
     isRedacted: boolean;
+
+    @Column({
+        type: 'timestamp',
+        nullable: true,
+        default: new Date(),
```

Comment:
I am not sure about default. When the column is created everything should be NULL, but on writing current timestamp is right

Reply:
Ah, maybe we should set it to the timestamp that was generated in the migration, otherwise TypeORM might generate a new migration each time ... (haven't checked this though)